Inspicio/Controllers/ImagesController.cs

```diff
@@ -31,20 +31,42 @@ public ImagesController(ApplicationDbContext _context, IHostingEnvironment _envi
         }
 
         // GET: Images
+        public class ImageDataModel
```

Comment:
The things that come out of (and go into) controller actions are often called ViewModels (it's a Model for the View).